desktop/src/main/java/bisq/desktop/main/portfolio/openoffer/OpenOffersView.java

```diff
@@ -116,6 +118,10 @@ public void initialize() {
         priceColumn.setComparator(Comparator.comparing(o -> o.getOffer().getPrice(), Comparator.nullsFirst(Comparator.naturalOrder())));
         volumeColumn.setComparator(Comparator.comparing(o -> o.getOffer().getVolume(), Comparator.nullsFirst(Comparator.naturalOrder())));
         dateColumn.setComparator(Comparator.comparing(o -> o.getOffer().getDate()));
+        paymentMethodColumn.setComparator(Comparator.comparing(
+                o -> o.getOffer() != null ? o.getOffer().getPaymentMethod().getId() : null,
```

Comment:
Is the null check necessary? The above calls to result of `o.getOffer()` don't null check.

Reply:
I agree, and will modify to bring it into line with the code above it.

Just to note, this line was copied from the existing PendingTradesView.java in which there is a payment method column.  Should I also make a change to do the same in PendingTradesView.java or would that need to be a separate code-cleanup type PR?

Reply:
I think it's fine if you leave the other one as is.